starter-code/pizza.js

```diff
@@ -1 +1,141 @@
+            totalPrice += 1;
+            console.log(totalPrice);
+            $('aside strong').text("$"+totalPrice);
+        }
+        
+    });
+
+    // Click in the mushrooms
+    $(".btn-mushrooms").click(function() {
+    
+        // Find every mushroom
+        if($(this).hasClass('active') === true){
+            //Delete the class
+            $('.mushroom').css('display', 'none');
+            $(this).removeClass('active');
+            $('aside ul li:nth-child(2)').css('display', 'none');
+            totalPrice -= 1;
+            console.log(totalPrice);
+            $('aside strong').text("$"+totalPrice);
+
+        } else {
+            $('.mushroom').css('display', 'block');
+            $(this).addClass('active');
+            $('aside ul li:nth-child(2)').css('display', 'block');
+            totalPrice += 1;
+            console.log(totalPrice);
+            $('aside strong').text("$"+totalPrice);
+        }
+    
+    });
+
+    // Click in the green-peppers
+    $(".btn-green-peppers").click(function() {
+    
+        // Find every green-pepper
+        if($(this).hasClass('active') === true){
+            //Delete the class
+            $('.green-pepper').css('display', 'none');
+            $(this).removeClass('active');
+            $('aside ul li:nth-child(3)').css('display', 'none');
+            totalPrice -= 1;
+            console.log(totalPrice);
+            $('aside strong').text("$"+totalPrice);
+
+        } else {
+            $('.green-pepper').css('display', 'block');
+            $(this).addClass('active');
+            $('aside ul li:nth-child(3)').css('display', 'block');
+            totalPrice += 1;
+            console.log(totalPrice);
+            $('aside strong').text("$"+totalPrice);
+        }
+    
+    });
+
+
+    // Click in the green-peppers
+    $('.btn-sauce').removeClass('active');
+    $('aside ul li:nth-child(4)').css('display', 'none');
+
+    $(".btn-sauce").click(function() {
+        
+        // Find every pepperonni
+        if($(this).hasClass('active') === true){
+            //Delete the class
+            $(this).removeClass('active');
+            console.log('sauce desactivated');
+            $('aside ul li:nth-child(4)').css('display', 'none');
+            totalPrice -= 3;
+            console.log(totalPrice);
+            $('aside strong').text("$"+totalPrice);
+        } else {
+            $(this).addClass('active');
+            console.log('sauce activated');
+            $('aside ul li:nth-child(4)').css('display', 'block');
+            totalPrice += 3;
+            console.log(totalPrice);
+            $('aside strong').text("$"+totalPrice);
+
+        }
+
+    });
+
+    $('.btn-crust').removeClass('active');
+    $('aside ul li:nth-child(5)').css('display', 'none');
+
+    $(".btn-crust").click(function() {
+        
+        // Find every pepperonni
+        if($(this).hasClass('active') === true){
```

Comment:
You only missed to add a toggleClass() for the crust and sauce for it to be fully working. Also be careful with copy pasting code and the comments